packages/tools/src/store/addEnabledElement.ts

```diff
@@ -13,6 +13,7 @@ import {
   touchToolEventDispatcher,
   keyboardToolEventDispatcher,
   imageSpacingCalibratedEventDispatcher,
+  annotationInterpolationEventDispatcher,
```

Comment:
Can we call this annotationGenerationEventDispatcher please?  Then, it will work for other types of generations, such as single click/call remote web server for AI generation/response with auto-generated events - possibly on multiple series.